package.json

```diff
@@ -46,6 +47,7 @@
         "bundle-loader": "^0.5.5",
         "classnames": "^2.2.5",
         "classnames-loader": "^2.0.0",
+        "concurrently": "^4.1.0",
```

Comment:
When you checkout to `develop`, you see, that we install all packages with `--save-exact` flag.